tsconfig.json

```diff
@@ -17,10 +17,19 @@
     "lib": [
       "es2018",
       "dom"
-    ]
+    ],
+    "paths": {
+      "hee-shared": [
+        "dist/hee-shared/hee-shared",
```

Comment:
is this path correct?

Reply:
yes its the build path for compilation/tree shaking

Reply:
cool